README.md

```diff
@@ -4,17 +4,21 @@ Flatpak for [SABnzbd](https://github.com/sabnzbd/sabnzbd).
 
 ## Build from source
 
+### Update the manifest
+1. Update the url for SABnzbd in the [manifest file](https://github.com/flathub/org.sabnzbd.sabnzbd/blob/85a8c16e7c2e351077c5cd83437ca4e43b2eb274/org.sabnzbd.sabnzbd.yaml#L117)
+2. Update the SHA256 hash for the SABnzbd source file
+
 ### Generating Python dependencies
 
 1. Install <https://github.com/flatpak/flatpak-builder-tools/tree/master/pip>
-2. Compare and adjust packages with <https://github.com/sabnzbd/sabnzbd/blob/4.4.1/requirements.txt>
+2. Compare and adjust packages with https://github.com/sabnzbd/sabnzbd/blob/master/requirements.txt
```

Comment:
It needs to be a fixed version, the upstream master branch doesn't always work for a version.